server/inspect.go

```diff
@@ -0,0 +1,51 @@
+package server
+
+import (
+	"encoding/json"
+	"fmt"
+	"net/http"
+
+	"github.com/go-zoo/bone"
+)
+
+type ContainerInfo struct {
+	Pid         int               `json:"pid"`
+	Image       string            `json:"image"`
+	CreatedTime int64             `json:"created_time"`
+	Labels      map[string]string `json:"labels"`
+}
+
+func (s *Server) StartInspectEndpoint() error {
+	mux := bone.New()
+
+	mux.Get("/containers/:id", http.HandlerFunc(func(w http.ResponseWriter, req *http.Request) {
+		containerId := bone.GetValue(req, "id")
+		ctr := s.GetContainer(containerId)
+		if ctr == nil {
+			http.Error(w, fmt.Sprintf("container with id: %s not found", containerId), http.StatusNotFound)
+			return
+		}
+		ctrState := ctr.State()
+		if ctrState == nil {
+			http.Error(w, fmt.Sprintf("container %s state is nil", containerId), http.StatusNotFound)
+			return
+		}
+
+		ci := ContainerInfo{
+			Pid:         ctrState.Pid,
+			Image:       ctr.Image(),
+			CreatedTime: ctrState.Created.UnixNano(),
+			Labels:      ctr.Labels(),
+		}
+		js, err := json.Marshal(ci)
+		if err != nil {
+			http.Error(w, err.Error(), http.StatusInternalServerError)
+			return
+		}
+		w.Header().Set("Content-Type", "applicaton/json")
+		w.Write(js)
+	}))
+
+	// TODO: Make this configurable
+	return http.ListenAndServe(":7373", mux)
```

Comment:
is this defaulting to listen just on localhost?

Reply:
localhost:7373/containers/id should work for this.

Reply:
Yeah, let's just make sure we don't expose anything (and afaict it just binds on localhost)

Reply:
@rhatdan I agree it should be restricted to localhost. Let me double check and ensure that is the case. Thanks!

Reply:
okay, it was listening on all interface so I pushed an update to explicitly specify localhost:7373.